R/post_bug.R

```diff
@@ -45,14 +46,14 @@ post_bug <- function(text, title, component, ...,
         return()
     }
     ask_final_confirmation()
-    url <- paste0(host, "rest/bug")
-    # bugs <- httr::POST(url,
-    #                    description = text,
-    #                    product = product,
-    #                    component = component,
-    #                    summary = title,
-    #                    version = version,
-    #                    ...,
-    #                    headers)
-    # bugs <- httr::content(bugs)
+    url <- paste0(host, "rest/bug?")
+    data <- list(
+        product = product,
+        component = component,
+        version = version,
+        summary = title
+    )
+    bugs <- httr::POST(url, body = data, encode = "json")
+    bugs <- httr::content(bugs)
+    print(bugs)
```

Comment:
Print is not a function to be used within a function. If you want to provide feedback to the user use the `return(bugs)`